app/controllers/point_details_controller.rb

```diff
@@ -5,7 +5,11 @@ class PointDetailsController < ApplicationController
 
   def update
     point_detail_service = CreateOrUpdatePointDetail.new(@user, @principle, @evaluation)
-    @point_detail, @point_details = point_detail_service.call(params[:point])
+    begin
+      @point_detail, @point_details = point_detail_service.call(params[:point])
+    rescue ActiveRecord::RecordInvalid
+      head(:unprocessable_entity)
+    end
```

Comment:
It can be made a bit more compact.

```suggestion
    @point_detail, @point_details = point_detail_service.call(params[:point])
  rescue ActiveRecord::RecordInvalid
    head(:unprocessable_entity)
```